src/components/CounterPage/Counter/index.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import './style.css';
+
+const Counter = ({ countValue, isEven, handleIncrease, handleDecrease, handleReset }) => {
+
+  return (
+
+    <div className='counterArea'>
+      <div className='numberArea'>{countValue}</div>
+      <div className={isEven ? 'parityAreaOdd' : 'parityAreaEven'}>{isEven ? 'Odd' : 'Even'}</div>
```

Comment:
Read this line attentively.  
Here your logic is:   If the number Even show Odd and if it's not Even.
Mistake in logic chain. You have to replace your logic to `isEven ? 'Even' : 'Odd'`. 
The same in the className area.
Check this link to revise the material https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Operators/%D0%A3%D1%81%D0%BB%D0%BE%D0%B2%D0%BD%D1%8B%D0%B9_%D0%BE%D0%BF%D0%B5%D1%80%D0%B0%D1%82%D0%BE%D1%80